src/kudu/consensus/raft_consensus.cc

```diff
@@ -3037,7 +3038,9 @@ Status RaftConsensus::RefreshConsensusQueueAndPeersUnlocked() {
   queue_->SetLeaderMode(pending_->GetCommittedIndex(),
                         CurrentTermUnlocked(),
                         active_config);
-  RETURN_NOT_OK(peer_manager_->UpdateRaftConfig(active_config));
+  RETURN_NOT_OK(peer_manager_->UpdateRaftConfig(
+        active_config,
+        queue_->GetQuorumMode()));
```

Comment:
this seems to be a remnant due to the PR splitting, where in PeerManager you needed quorum mode

Reply:
Yes. Fixed it and running another round of tests and benchmarks before merging